pyxem/generators/subpixelrefinement_generator.py

```diff
@@ -113,6 +113,23 @@ def center_of_mass_method(self, square_size):
         This will work poorly on disks with strong dynamic contrast
         """
 
+        def _center_of_mass_hs(z):
```

Comment:
Can we have some docstrings for these in the standard format please?

Reply:
Even for secreted internal functions?(!)

Reply:
Yes.